lib/LanguageServerReferenceFinder/Handler/GotoDefinitionHandler.php

```diff
@@ -31,11 +32,21 @@ class GotoDefinitionHandler implements Handler, CanRegisterCapabilities
      */
     private $locationConverter;
 
-    public function __construct(Workspace $workspace, DefinitionLocator $definitionLocator, LocationConverter $locationConverter)
-    {
+    /**
+     * @var ClientCapabilities
+     */
+    private ClientCapabilities $clientCapabilities;
```

Comment:
I forgotten that the whole project still depends on (unsupported) PHP 7.3 and typed properties are not allowed.

Reply:
yeah, it would be good to bump it again. just a big PITA with 50 packages.